### PR TITLE
feat: upgrade to jest 27

### PR DESCRIPTION
**Pull request by bradzacher**
Jest v27 has been released and with it a new major for jest-snapshot.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,14 +15,14 @@
     "cacheDirectory": "./.jest/cache"
   },
   "dependencies": {
-    "jest-snapshot": "^26.3.0"
+    "jest-snapshot": "^27.0.2"
```
Comment (igor-dv) [inline package.json:18]: should the peerDependencies also be changed then ?

Comment (bradzacher) [reply package.json:18]: definitely should, yeah!
Regardless this will be a technically breaking change due to the major bump

Comment (igor-dv) [reply package.json:18]: yep 👍 , I will bump the major version

**Review by igor-dv — APPROVED** [review]

**Comment by igor-dv** [comment]
Released as v5
Thank you @bradzacher !

**Pull request merged**